feed-notifier.php

```diff
@@ -160,6 +160,25 @@ function deliver_mail_feed ($feed_url) {
 
 }
 
+function delMail($res){
+  if ($res) 
+  { 
+    echo "ok";
+  } else {
+    echo "false";
+  }
+
+  $files = file_get_contents($res);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "$files" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1481)